README.md

```diff
@@ -102,6 +102,7 @@ Validator                               | Description
 **isBoolean(str [, options])**          | check if the string is a boolean.<br/>`options` is an object which defaults to `{ loose: false }`. If `loose` is is set to false, the validator will strictly match ['true', 'false', '0', '1']. If `loose` is set to true, the validator will also match 'yes', 'no', and will match a valid boolean string of any case. (e.g.: ['true', 'True', 'TRUE']).
 **isBtcAddress(str)**            | check if the string is a valid BTC address.
 **isByteLength(str [, options])**          | check if the string's length (in UTF-8 bytes) falls in a range.<br/><br/>`options` is an object which defaults to `{ min: 0, max: undefined }`.
+**isColor(str [, options])**                  | check if the string is a css compatible color.<br/><br/> `options` is an optional object that can be supplied with `format` key with a value of css compatible format (hex, hexa, rgb, rgba, hsl, hsla).
```

Comment:
Do we want to remain doing something with `hexa`, `rgba` and `hsla`? It's not in the code right now, but might be nice to be able to specify in format